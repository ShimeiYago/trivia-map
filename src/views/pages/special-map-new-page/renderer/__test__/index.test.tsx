import { shallow, ShallowWrapper } from 'enzyme';
import { Renderer, State } from '..';

let wrapper: ShallowWrapper<null, State, Renderer>;

describe('Shallow Snapshot Tests', () => {
  beforeEach(() => {
    wrapper = shallow(<Renderer />);
  });

  it('basic', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('redirect', () => {
    wrapper.setState({
      specialMapIdForRedirect: 1,
    });
    expect(wrapper).toMatchSnapshot();
  });
});
