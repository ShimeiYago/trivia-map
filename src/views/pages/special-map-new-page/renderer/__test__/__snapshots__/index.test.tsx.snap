// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shallow Snapshot Tests basic 1`] = `
<GlobalMenu
  topBarPosition="static"
>
  xxx
</GlobalMenu>
`;

exports[`Shallow Snapshot Tests redirect 1`] = `
<Navigate
  to="/special-maps/edit/1"
/>
`;
